content/webapp/pages/exhibitions/[exhibitionId]/visual-stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as prismic from '@prismicio/client';
+import VisualStory, {
+  returnVisualStoryProps,
+} from '@weco/content/pages/visual-stories/[visualStoryId]';
+import { getServerData } from '@weco/common/server-data';
+import { looksLikePrismicId } from '@weco/common/services/prismic';
+import { createClient } from '@weco/content/services/prismic/fetch';
+import {
+  fetchVisualStories,
+  fetchVisualStory,
+} from '@weco/content/services/prismic/fetch/visual-stories';
+import { setCacheControl } from '@weco/content/utils/setCacheControl';
+
+export const getServerSideProps = async context => {
+  setCacheControl(context.res);
+  const client = createClient(context);
+  const serverData = await getServerData(context);
+  if (!serverData?.toggles?.visualStories?.value) {
+    return { notFound: true };
+  }
+
+  const visualStoriesQuery = await fetchVisualStories(client, {
+    filters: [
+      prismic.filter.at(
+        'my.visual-stories.related-exhibition',
+        context.query.exhibitionId
+      ),
+    ],
+  });
+
+  if (visualStoriesQuery.results.length === 0) {
+    return { notFound: true };
+  }
+
+  // We are assuming for now that there is only one VS per exhibition
+  const visualStoryId = visualStoriesQuery.results[0].id;
+
+  if (!looksLikePrismicId(visualStoryId)) {
+    return { notFound: true };
+  }
+
+  const visualStoryDocument = await fetchVisualStory(client, visualStoryId);
```

Comment:
Do we need to fetch this again? as we've already fetched the visual story with the matching exhibition id above

Reply:
That's a really good point -  and it's made me realise I forgot to add `return` to the function below!